dxr/templates/nunjucks/promoted.html

```diff
@@ -0,0 +1,27 @@
+{% macro promoted_list(query, promoted, promoted_count, www_root, tree) -%}
+  {% for result in promoted %}
+    <div class="result-head">
+      <div class="left-column">
+        <div class="icon-container {{ result.iconClass }}"></div>
+      </div>
+      <div>
+        {{ result.pathLine | safe }}
+      </div>
+    </div>
+  {% endfor %}
+{%- endmacro %}
+
+{%- macro promoted_header(promoted_query, promoted_count, promoted_count_formatted, top_of_tree, tree) -%}
+  <p class="promoted-see-more">
+    {% if promoted_count == 1 %}
+      Found 1 path result:
```

Comment:
Let's get the relationship straight between this caption and the main "2 results from the code tree:" one:

1. Right now, there's this weird interleaving where it says "2 results:", then there's the promoted box, then finally the results show up. 
![screen shot 2016-07-22 at 16 34 42](https://cloud.githubusercontent.com/assets/295816/17070510/4f619926-502a-11e6-9d61-a4a42e7ba956.png)
 This used to make sense when the promoted stuff was on the right.
2. We might as well homogenize the wording, changing this to "1 path result". There's no need to say "Found."